web-wallet/CHANGELOG.md

```diff
@@ -10,16 +10,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add visible version, commit hash and build date [#1441]
+- Add Address validation (Transfer flow) [#1377]
 
 ### Changed
 
+- Change Get Quote API Endpoint to env variable [#1311]
+
 ### Removed
 
 - Remove the use of `checkValidity()` in Send and Stake flow amounts validity checks [#1391]
 
 ### Fixed
 
-- Fix typo in routes/welcome/__tests__/page.spec.js [#1445]
+- Fix typo in routes/welcome/tests/page.spec.js [#1445]
```

Comment:
You could have wrapped the real name in backticks instead.

Reply:
Should have been escaped with "\_\_", it's done now.